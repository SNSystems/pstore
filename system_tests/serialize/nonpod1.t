# %binaries = the directories containing the executable binaries
# %T = the test output directory
# %S = the test source directory
REQUIRES: examples
RUN: %binaries/example_nonpod1 > %T/nonpod1_actual.txt
RUN: diff %T/nonpod1_actual.txt %S/nonpod1_expected.txt
