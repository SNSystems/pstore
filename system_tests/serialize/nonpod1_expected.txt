Writing: foo(42)
Wrote these bytes: 2a 00 00 00
Read: foo(42)
